app/services/fuzzySearchService.js

```diff
@@ -46,9 +46,11 @@ const getSortedResults = (records, sortBy) => {
   return records
 }
 
-const applyFilters = (searchResults, { schemes = [], amounts = [] }) => {
+const applyFilters = (searchResults, { schemes = [], counties = [], amounts = [] }) => {
```

Comment:
You need to add a field called counties to the filterBy object in the paymentData endpoint's payload as well, as otherwise the endpoint will reject the request. Please check paymentdata.js for how the amounts and schemes have been added to the validate object.
Furthermore, please try and add appropriate integration tests for the paymentData endpoint, using the counties field.

Reply:
good point, Added counties parameter to /paymentdata api